app/Pain/Pain/Interface/Camera/RectangleDetectionViewController.swift

```diff
@@ -168,19 +165,69 @@ class RectangleDetectionViewController: CameraViewController {
     }
     
     private func filterRectangles(rectangles: [Detection]) -> [Detection] {
-        var rectangles = filterRectanglesByBounds(rectangles: rectangles)
-        rectangles = filterRectanglesByInclusion(rectangles: rectangles)
+        let rectangles = filterRectanglesByBounds(rectangles: rectangles)
+//        rectangles = filterRectanglesByInclusion(rectangles: rectangles)
         return rectangles
     }
     
     private func filterRectanglesByBounds(rectangles: [Detection]) -> [Detection] {
```

Comment:
Either rename the method to not contain Rectangles in the name, or make the argument nameless.
```swift
filterByBounds(rectangles: rectangles)
```
or
```swift
filterRectanglesByBounds(rectangles)

func filterRectanglesByBounds(_ rectangles: [Detection]) -> [Detection]
```

Reply:
I renamed some things